docs/archive_data_migration_to_berkeley.md

```diff
@@ -0,0 +1,470 @@
+# Migrating Mainnet Database to Berkeley
+
+Before you start planning the migration of your Mainnet database into the Berkeley version of the Mina network, read this document in entirety.
+
+## Migration apps
+
+Two applications are required to migrate the Mainnet database into the Berkeley version of the Mina network. 
+
+These applications are shipped by MF/O1 Labs:
+
+- **berkeley-migration**
+
+  The berkeley-migration application migrates as much data as possible from the Mainnet database and downloads precomputed blocks to get the window density data. 
+  
+  This application runs against the Mainnet and the new database.
+
+- **replayer app in migration mode**
+
+  The replayer app in migration mode replays the transactions in the partially migrated database and populates the `accounts_accessed` and `accounts_created` tables. This application also performs the checks performed by the standard replayer, except that ledger hashes are not checked because the hard fork ledger has greater depth, which results in different hashes. This application runs only against the new database.
```

Comment:
So, we use the replayer application twice in the migration. 

First: run in migration mode against new database.
Second: Run the replayer application against your existing Mainnet database to verify the blockchain state.

Seems like we can mention here